integration/test_auth.py

```diff
@@ -577,6 +600,19 @@ def test_verify_id_token_revoked(new_user, api_key):
     claims = auth.verify_id_token(id_token, check_revoked=True)
     assert claims['iat'] * 1000 >= user.tokens_valid_after_timestamp
 
+def test_verify_id_token_tolerance(new_user, api_key):
+    expired_id_token = _sign_in_with_password(new_user.email, 'password', api_key)
```

Comment:
Should this be `new_user_with_params` here?

I'm guessing it's passing `None` to get an invalid token which is expired by default. Is this the only way to obtain an expired `id_token`?

Reply:
I'm honestly not sure -- I couldn't find any integration examples with expired id tokens

Reply:
yep, lints failed on this test, so deleting this actually fixes that as well!

Reply:
`test_session_cookies_with_tolerance`, `test_verify_session_cookie_tolerance` and `test_verify_session_cookie_clock_skew_seconds_range` should also be dropped for the same reason. Thank you!